### ISSUE TITLE
Le projet est il abandonné en 2024 ? 

### ISSUE THREAD

**Issue by NS-Riders** [openairlinetycoon/OpenATDeluxe#32]
Bonjour, je viens vers vous pour savoir si le projet Airlines Tycoon était abandonné en 2024  ?
Car je vois que depuis quelques années il n'y a plus de contribution malheureusement et j'ai de plus en plus de mal à faire tourner le jeu original sur Steam.
Merci d'avance

**Comment by mygrexit** [comment]
Yes, it has been abandoned, like many great open source projects.
